analyzer/src/main/resources/python_compatibility.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/python
```

Comment:
If this script is always supposed to be run with Python 3, the shebang line should also say `#!/usr/bin/python3`, I guess.

Reply:
- [x] Improve shebang line.